packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/siriusweb.graphqls

```diff
@@ -40,6 +41,24 @@ type ProjectTemplate {
   imageURL: String!
 }
 
+type ViewerLibrariesConnection {
+  edges: [ViewerLibrariesEdge!]!
+  pageInfo: PageInfo!
+}
+
+type ViewerLibrariesEdge {
+  node: Library!
+}
+
+type Library {
+  id: ID!
+  projectId: String!
+  name: String!
+  content: String!
```

Comment:
What is this `content` field? Are you giving the raw data of the library to the frontend?